randovania/lib/random_lib.py

```diff
@@ -48,12 +44,24 @@ def iterate_with_weights(
         yield pickup_node
```

Comment:
while you're in here, wanna rename `pickup_node` and `action` into generic names?

Reply:
oh yeah oops